src/components/reservation/helper.ts

```diff
@@ -149,4 +150,21 @@ export const getAuthorLine = (
     : [t("materialHeaderAuthorByText"), author, year].join(" ");
 };
 
+export const getManifestationsToReserve = (
+  reservableManifestations: Manifestation[] | null,
+  isPeriodical?: boolean
+) => {
+  if (isPeriodical) {
+    return reservableManifestations;
+  }
+  if (!reservableManifestations || reservableManifestations.length < 1) {
+    return null;
+  }
+  // If the work isn't fictional we only want to reserve the latest manifestation.
+  if (!materialIsFiction(reservableManifestations[0])) {
+    return [getLatestManifestation(reservableManifestations)];
```

Comment:
Another approach to this would be `orderManifestationsByYear(reservableManifestations, "desc").slice(0,1)`. This would also handle a empty `reservableManifestations` quite elegantly. It is not as urgent now as we explicitly check on L160.

Reply:
I think it's more readable as it is currently.